src/hypertable.h

```diff
@@ -9,6 +9,9 @@
 #include "tablespace.h"
 #include "scanner.h"
 
+#define OLD_INSERT_BLOCKER_NAME	"insert_blocker"
+#define INSERT_BLOCKER_NAME "ts_insert_blocker_on_hypertable"
```

Comment:
not sure about suffix here. Isn't it implicit that this in on a hypertable, since it is a hypertable?

Reply:
Changed to just `ts_insert_blocker`.